Pipeline.md

```diff
@@ -0,0 +1,40 @@
+# Pipeline
+
+Coconut’s pipe operators allow for pipeline-style function application, which makes multiple function calls more readable.
+ 
+The different operators are:
+```python
+(|>)    => pipe forward
+(|*>)   => multiple-argument pipe forward
+(<|)    => pipe backward
+(<*|)   => multiple-argument pipe backward
+```
+*Note: refer to the [documentation](http://coconut.readthedocs.io/en/master/DOCS.html#operators) for information about operator precedence and associativity.*
+
+<br/>
+
+Here's how function calls are structured in Coconut using pipes:
+```python
+obj |> .attribute |> .method(args) |> func$(args) |> .[index]
+```
+
+Compare that to equivalent Python code:
+```python
+func(args, obj.attribute.method(args))[index]
```

Comment:
The Python code here looks simpler than the Coconut code and I don't know what to do about it...

Reply:
Not necessarily a bad thing.  Part of using a tool is also knowing when to use an alternative as not everything can be perfect.  The tutorial can also explain what might be easier in Python.

Reply:
I think that it's more of an idea of how the attributes or calls get ordered as opposed to how visually compact it looks - when I read this in the original context, I think what stood out the most was how the order in which things are called seems a little more intuitive and sensible than using nested method and function calls.